inference-engine/src/vpu/graph_transformer/src/middleend/allocator/allocator.cpp

```diff
@@ -480,10 +469,6 @@ std::size_t Allocator::freeCMXMemoryAmount() const {
     return _maxCmxSize - offset;
 }
 
-std::size_t Allocator::freeMemoryAmount(const MemoryType& type) const {
-    return type == MemoryType::CMX ? freeCMXMemoryAmount() : freeDDRMemoryAmount();
```

Comment:
Not sure about removing CMX check. CMX has the same capacity both for ma2480/ma2085